helper/token.js

```diff
@@ -0,0 +1,79 @@
+const promise = require('bluebird');
+const dbHelper = require('../helper/database');
+const request = require('request');
+const tokenCollection = 'token';
+const canvasUrl = 'http://canvas.differ.chat/api/v1/users/36/courses?access_token=';
```

Comment:
Done, in this branch as refactoring as suggested for the canvas service changes needed to implement :)